Lsof.8

```diff
@@ -1676,11 +1676,20 @@ FAQ (The \fBFAQ\fP section gives its location.)
 On Linux this option also prints the state of UNIX domain sockets.
 .TP \w'names'u+4
 .B \-t
-specifies that
-.I lsof
-should produce terse output with process identifiers only and no header \-
-e.g., so that the output may be piped to
-.IR xargs kill.
+produce terse output comprising only process identifiers (without a
+header), so that it is easy to use programmatically. e.g.
+.nf
+
+	# reload anything using old SSL
+	lsof -t /lib/*/libssl.so.* | xargs -r kill -HUP
+
+        # get list of processes and then iterate over them
```

Comment:
It seems that one extra white space is at the start of the line. Could you remove it for aligning.